### ISSUE TITLE
Will this work with tensorflow 1.15?

### ISSUE THREAD

**Issue by akshayacharya97** [matterport/Mask_RCNN#2596]
i AM USING JETSON XAVIER nx. I usually use tensorflow 1.8 and keras 2.1.5 for geting this repository to work on aws inside jupyter notebook. No other version of tensorflow seems to wrok. Now I have had to switch to the jetson xavire, I was wondering if tensorflow 1.15 would be compatible or not. The jetson doesnt have a version of tensorflow older than 1.15. What can i do?

**Comment by alexander-pv** [comment]
Hi,  I updated Mask-RCNN for tensorflow v2 and added manual about model optimization on Xavier AGX. I believe that it can be useful for your work with Xavier NX.
You can see fork here: [link](https://github.com/alexander-pv/maskrcnn_tf2)
For now, COCO weights are not included, only ImageNet weights for backbones by default.
